src/slot_migrate.h

```diff
@@ -145,6 +145,13 @@ class SlotMigrate : public Redis::Database {
   };
   ParserState stat_ = ArrayLen;
 
+  enum ThreadState {
```

Comment:
`enum class` is better?

Reply:
Sure, it makes sense. Please take a look again